advanced/01-pihole.conf

```diff
@@ -18,8 +18,6 @@
 #                    WITHIN /etc/dnsmasq.d/yourname.conf                      #
 ###############################################################################
 
-addn-hosts=/etc/pihole/gravity.list
-addn-hosts=/etc/pihole/black.list
 addn-hosts=/etc/pihole/local.list
```

Comment:
Perhaps in the future we can also remove this list, as it can be easily determined by FTL (only requires getting the IP addresses and the hostname)

Reply:
This is a good point. We will surely come back to it when the gravity script is seeing the next overhauling.